security/cmd/istio_ca/main.go

```diff
@@ -197,9 +194,6 @@ func init() {
 	flags.IntVar(&opts.grpcPort, "grpc-port", 8060, "The port number for Citadel GRPC server. "+
 		"If unspecified, Citadel will not serve GRPC requests.")
 
-	flags.StringVar(&opts.grpcHostname, "grpc-hostname", "istio-ca", "deprecated")
```

Comment:
Not sure if it is safe to remove flags - the binary may be used in other environments ( standalone, etc ) - it is safer to mark it as deprecated, but leave it for few version (maybe with a warning if it is used )